src/ui/components/SidePanel.tsx

```diff
@@ -65,17 +87,27 @@ export default function SidePanel() {
   }
 
   return (
-    <div
-      className="w-full overflow-hidden rounded-lg bg-bodyBgcolor text-xs"
-      data-test-id="leftSidebar"
-    >
-      {selectedPrimaryPanel === "explorer" && <PrimaryPanes />}
-      {selectedPrimaryPanel === "debugger" && <SecondaryPanes />}
-      {selectedPrimaryPanel === "comments" && <CommentCardsList />}
-      {selectedPrimaryPanel === "events" && <EventsPane items={items} />}
-      {selectedPrimaryPanel === "cypress" && <TestSuitePanel />}
-      {selectedPrimaryPanel === "protocol" && <ProtocolViewer />}
-      {selectedPrimaryPanel === "search" && <SearchFilesReduxAdapter />}
+    <div className="flex w-full flex-col gap-2">
+      {shouldShowDevToolsNag(nags, viewMode) && (
+        <div className={styles.TourBox}>
+          <h2>Welcome to Replay!</h2>
+          <p>To get started, click into DevTools so we can show off some time travel features!</p>
+          <button type="button" onClick={() => handleToggle("dev")} style={{ padding: "5px 12px" }}>
+            <div>Open DevTools</div>
+            <MaterialIcon style={{ fontSize: "16px" }}>arrow_forward</MaterialIcon>
+          </button>
+        </div>
+      )}
+
+      <div className="h-full w-full overflow-hidden rounded-lg bg-bodyBgcolor text-xs">
```

Comment:
Nit: I prefer to use `h-0 flex-grow` instead of `h-full`. `h-full` works because `flex-shrink` defaults to `1` but I prefer the explicit "no natural height but grow to fill the space" vs "try to be 100% but you can shrink"